nixos/modules/programs/fish.nix

```diff
@@ -101,6 +101,9 @@ in
       end
     '';
 
+    # include programs that bring their own completions
+    environment.pathsToLink = [ "/share/fish/vendor_completions.d" ];
```

Comment:
I believe this error only occurs if you have no packages containing `/share/fish/vendor_completions.d`, like me. When I add a package that contains that, like `docker`, then I do not get the above error.